nixos/modules/services/web-apps/pihole-adminlte.nix

```diff
@@ -0,0 +1,127 @@
+{ pkgs
+, lib
+, config
+, ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.pihole-adminlte;
+  ftlCfg = config.services.pihole-ftl;
+  user = ftlCfg.user;
+  group = ftlCfg.group;
```

Comment:
Aliasing user and group globally seems a bit dangerous as they are quite generic